.github/workflows/moodle-plugin-ci.yml

```diff
@@ -31,8 +31,8 @@ jobs:
       fail-fast: false
       matrix:
         php: ['8.1']
-        moodle-branch: ['MOODLE_401_STABLE', 'MOODLE_402_STABLE']
-        database: [pgsql, mariadb]
+        moodle-branch: ['MOODLE_402_STABLE'] # ['MOODLE_401_STABLE', 'MOODLE_402_STABLE'] # TODO: REENABLE BEFORE PR
+        database: [pgsql] # [pgsql, mariadb] # TODO: REENABLE BEFORE PR
```

Comment:
this is the big thing we need a call on. Do we enable matrices for additional DB and moodle version?